Docker/LaunchRViz.bash.ubuntu-galactic

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
I suggest changing the file name:
- This actually launches ROS 2 navigation stack
- RViz2 is just an addition to the stack

Reply:
LaunchNavStack?